apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -113,110 +121,187 @@ const NewProject = () => {
+              </ScrollableList>
+
+            </div>
+          )
+        }
+
+        if (loading) return (
+          <RepoInfo>
+            <LoadingAnimation />
+            <div>Loading repositories...</div>
+          </RepoInfo>
+        )
+
+        if (error) return (
+          <RepoInfo>
+            <Text size="xsmall" style={{ margin: '20px 0' }}>
+              Error {JSON.stringify(error)}
+            </Text>
+            <Button wide mode="strong" onClick={() => refetch()}>
+              Try refetching?
+            </Button>
+          </RepoInfo>
+        )
+      }}
+    </Query>
+  )
 
-                    <ScrollableList>
-                      <RepoList visibleRepos={visibleRepos} repoArray={repoArray} />
-                    </ScrollableList>
-
-                    <Info css={`margin: ${3 * GU}px 0`}>
-                        Projects in Aragon are a one-to-one mapping to a GitHub repository.
-                        You’ll be able to prioritize your backlog, reach consensus on issue
-                        valuations, and allocate bounties to multiple issues.
-                    </Info>
-
-                    <Button
-                      mode="strong"
-                      wide
-                      onClick={handleNewProject}
-                      disabled={repoSelected < 0}
-                    >
-                      Submit
-                    </Button>
-                  </div>
-                )
-              }
-
-              if (loading) return (
-                <RepoInfo>
-                  <LoadingAnimation />
-                  <div>Loading repositories...</div>
-                </RepoInfo>
-              )
-
-              if (error) return (
-                <RepoInfo>
-                  <Text size="xsmall" style={{ margin: '20px 0' }}>
-                      Error {JSON.stringify(error)}
-                  </Text>
-                  <Button wide mode="strong" onClick={() => refetch()}>
-                      Try refetching?
-                  </Button>
-                </RepoInfo>
-              )
-            }}
-          </Query>
+  return (
+    <>
+      {github.status !== 'authenticated' ? (
+        <div style={{ width: '100%', textAlign: 'center', marginBottom: (3 * GU) + 'px' }}>
+          <Button
+            mode="normal"
+            onClick={handleGithubSignIn}
+            icon={<IconGitHub
+              color={`${theme.surfaceContentSecondary}`}
+              width="16px"
+              height="16px"
+            />}
+            label="Select from GitHub"
+          />
         </div>
-      </div>
-    </React.Fragment>
+      ) : (
+        <RepoQuery />
+      )}
+      <Button
+        mode="strong"
+        wide
+        onClick={handleNewProject}
+        disabled={repoSelected < 0}
+      >
+        Submit
+      </Button>
+    </>
+  )
+}
+GitHubRepoList.propTypes = PropTypes.func.isRequired
+
+const createProject = () => {
+  console.log('creating native project')
```

Comment:
Ah, it looks like this PR is meant to be left unmerged until we implement the rest of this? 

Can we build this in a "feature flipped" way, so that we don't have to delay merging this PR for an extended period of time, but we also don't have unsupported elements in our UI? Ideally, we could do this with an environment variable, or maybe a URL param (like, add `?features=nativeprojects` to the end of the URL), but for an initial version I think we could even have something like `if (false)` in the code somewhere, to make it easy to toggle on & off in a development environment.

Reply:
It was meant to be merged at more or less the same time as the rest of it - but "feature flipped" makes sense as safety measure, to make sure there are no dangling threads accidentaly deployed where they shouldn't.
The easiest way in this particular case was to disable the -OR- part with a small `true?` statement - not very elegant, but it felt justified...